src/backend/distributed/planner/multi_router_planner.c

```diff
@@ -2407,6 +2407,13 @@ TargetShardIntervalForFastPathQuery(Query *query, Const **partitionValueConst,
 		PruneShards(relationId, relationIndex, make_ands_implicit((Expr *) quals),
 					&queryPartitionValueConst);
```

Comment:
This function could use some clarification. What is the difference between distributionKeyValue and queryPartitionValueConst?

Reply:
improved a bit, is it better now?